rust/src/wallet.rs

```diff
@@ -19,23 +18,37 @@ pub const TESTNET_ELECTRUM: &str = "ssl://blockstream.info:993";
 /// Wallet has to be managed by Rust as generics are not support by frb
 static WALLET: Storage<Mutex<Wallet>> = Storage::new();
 
+pub enum Network {
+    Mainnet,
+    Testnet,
+    Regtest,
```

Comment:
❓ Why are we adding regtest as enum option, if we do not support it?

Reply:
I was just converting the tests initially when I added it. I anticipate it being useful when testing lightning